src/components/create-curriculum-page-component/CreateCurriculumPageComponent.tsx

```diff
@@ -23,11 +23,12 @@ export class CreateCurriculumPageComponent extends React.Component<ICreateCurric
     constructor(props: any) {
         super(props)
         this.state = {
-            skillsToCurriculumArray: [[new Skill(0, '', new Category(0, ''))]],
+            skillsToCurriculumArray: [],
             newCurriculumName: '',
-            checkRender: [],
             filterSkillsMap: [new Skill(0, '', new Category(0, ''))],
-            filterSkillsORSkillsMap: Boolean
+            filterSkillsORSkillsMap: Boolean,
+            search: false,
+            colors: ['white', 'red', 'orange', 'yellow', 'green', 'blue', 'indigo', 'violet']
```

Comment:
this is fine for now, but colors should probably end up in the store eventually, or somewhere globally defined